README.md

```diff
@@ -128,6 +128,8 @@ redis-only-metrics     | REDIS_EXPORTER_REDIS_ONLY_METRICS    | Whether to also
 include-system-metrics | REDIS_EXPORTER_INCL_SYSTEM_METRICS   | Whether to include system metrics like `total_system_memory_bytes`, defaults to false.
 is-tile38              | REDIS_EXPORTER_IS_TILE38             | Whether to scrape Tile38 specific metrics, defaults to false.
 skip-tls-verification  | REDIS_EXPORTER_SKIP_TLS_VERIFICATION | Whether to to skip TLS verification
+tls-client-key-file    | REDIS_EXPORTER_TLS_CLIENT_KEY_FILE   | Full path to the client key file if the server requires TLS client authentication
```

Comment:
Question: your docs say `full path to the client key file`. Does that mean the expected parameter is a path (and the redis lib figures out what the file is) or is it the full name of the client key file?

Reply:
fyi, the text here is somewhat inconsistent with the `--help` output which says `Optional client certificate file...`

Reply:
It's the path including the file name which is passed to `tls.LoadX509KeyPair`. I should make the doc consistent with the flag description. Which is less ambiguous to you?

Reply:
Yeah, in that case maybe dbe a bit more explicit that it's a file name that's expected: 
e.g. `Name of the client key file (including full path) if the server requires TLS client authentication`

Reply:
Updated both the README and flag descriptions!

Reply:
Woooooooot !